package.json

```diff
@@ -15,10 +15,11 @@
     "semver"
   ],
   "scripts": {
-    "test": "npm run build && npm run lint && mocha -r ts-node/register test/index.spec.ts",
-    "build": "tsc",
+    "test": "npm run build && npm run lint && ./automation/test.js",
+    "build": "tsc && npm run docs",
     "lint": "tslint -t stylish -c tslint.json 'src/**/*.ts' 'test/**/*.ts'",
-    "docs": "npm run build && node generateDocs"
+    "docs": "./automation/docs.js",
+    "ci": "catch-uncommitted && npm test"
```

Comment:
`catch-uncommitted` needs to come _after_ the `npm test` - it now doesn't run the build, it just checks the state (so you can run the build however you like), so it needs to happen after all the build & docs & testing etc has happened.

Reply:
ah I missed that!

Reply:
To be fair, I did just silently change it from the previous script behaviour without mentioning anything. This one's definitely my fault :smiley:. Sorry!

Reply:
I moved things around and it should be good now